docs/FAQ.md

```diff
@@ -373,3 +373,38 @@ stripping out nested `vendor` directories.
 ## How do I configure a dependency that doesn't tag its releases?
 
 Add a constraint to `Gopkg.toml` that specifies `branch: "master"` (or whichever branch you need) in the `[[constraint]]` for that dependency. `dep ensure` will determine the current revision of your dependency's master branch, and place it in `Gopkg.lock` for you. See also: [What is the difference between Gopkg.toml and Gopkg.lock?](#what-is-the-difference-between-gopkgtoml-the-manifest-and-gopkglock-the-lock)
+
+## How do I use `dep` with Docker?
+
+`dep ensure -vendor-only` creates the vendor folder from a valid `Gopkg.toml` and `Gopkg.lock` without checking for Go code.
```

Comment:
Actually, `dep ensure -vendor-only` care about `Gopkg.lock` only. But it still requires `Gopkg.toml` to exists. :grin:

Reply:
Yeah that threw me off too for a sec but the FAQ is less about "how things are exactly implemented" and more high level, "you need both a toml and lock in order for this to work" type stuff. 😀 

I vote for leaving it as-is for now since both files are required.